src/main/kotlin/apply/application/ApplicationFormService.kt

```diff
@@ -1,63 +1,106 @@
-                submittedDateTime = createLocalDateTime(2019, 11, 5, 10),
-                recruitmentId = 1L,
-                applicantId = 1L,
-                answers = Answers(
-                    mutableListOf(
-                        Answer("고객에게 가치를 전달하고 싶습니다.", 1L),
-                        Answer("도전, 끈기", 2L)
-                    )
+                ApplicationForm(
+                        referenceUrl = "",
+                        submitted = true,
+                        createdDateTime = createLocalDateTime(2019, 10, 25, 10),
+                        modifiedDateTime = createLocalDateTime(2019, 11, 5, 10),
+                        submittedDateTime = createLocalDateTime(2019, 11, 5, 10),
+                        recruitmentId = 1L,
+                        applicantId = 1L,
+                        answers = Answers(
+                                mutableListOf(
+                                        Answer("고객에게 가치를 전달하고 싶습니다.", 1L),
+                                        Answer("도전, 끈기", 2L)
+                                )
+                        )
+                ),
+                ApplicationForm(
+                        referenceUrl = "https://www.google.com",
+                        submitted = true,
+                        createdDateTime = createLocalDateTime(2019, 10, 25, 10),
+                        modifiedDateTime = createLocalDateTime(2019, 11, 5, 10),
+                        submittedDateTime = createLocalDateTime(2019, 11, 5, 10),
+                        recruitmentId = 1L,
+                        applicantId = 2L,
+                        answers = Answers(
+                                mutableListOf(
+                                        Answer("스타트업을 하고 싶습니다.", 1L),
+                                        Answer("책임감", 2L)
+                                )
+                        )
                 )
-            ),
-            ApplicationForm(
-                referenceUrl = "https://www.google.com",
-                submitted = true,
-                createdDateTime = createLocalDateTime(2019, 10, 25, 10),
-                modifiedDateTime = createLocalDateTime(2019, 11, 5, 10),
-                submittedDateTime = createLocalDateTime(2019, 11, 5, 10),
-                recruitmentId = 1L,
-                applicantId = 2L,
-                answers = Answers(
-                    mutableListOf(
-                        Answer("스타트업을 하고 싶습니다.", 1L),
-                        Answer("책임감", 2L)
-                    )
-                )
-            )
         )
         applicationFormRepository.saveAll(applicationForms)
     }
+
+    fun save(applicantId: Long, recruitmentId: Long, applicationFormSaveRequest: ApplicationFormSaveRequest) {
+        if (applicationFormRepository.findByRecruitmentIdAndApplicantId(recruitmentId, applicantId) != null) {
+            throw IllegalAccessException("이미 저장된 지원서가 있습니다.")
+        }
```

Comment:
```suggestion
        applicationFormRepository.findByRecruitmentIdAndApplicantId(recruitmentId, applicantId)
            ?: throw IllegalArgumentException("이미 저장된 지원서가 있습니다.")
```

코틀린의 앨비스 연산자를 이용하여 위와 같이 수정해볼 수 있겠다는 생각이 드네요😎

Reply:
뭔가 이상하여 다시 확인해보니 null이 아닐 경우군요..!!

무시하셔도 되는 코멘트입니다😂